### PR TITLE
Reduce structure view

### PR DESCRIPTION
**Pull request by pb82**
## Motivation

Reduce the structure view in the schema editor to what we discussed in the UX review: no interactivity, only list the types and their properties. I suggest that we write a different component for the structure view in the resolver tab because it has a different organization and more features.

## Verification Steps

Paste this Schema and verify the types are rendered correctly:

```
  type Meme {
    id: ID! @isUnique
    photoUrl: String!
  }
  
  type Query {
    allMemes:[Meme!]!
  }
  
  type Mutation {
    createMeme(photoUrl: String!):Meme!
  }
  
  type Subscription {
    _: Boolean
  }
```

## Checklist:

- [x] Code has been tested locally by PR requester
- [ ] Changes have been successfully verified by another team member 

## Progress

- [x] Finished task

## Screenshot

bildschirmfoto 2018-07-26 um 18 36 46

### CODE REVIEW

File: ui/components/common/GraphQLFormatters.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+const formatType = type => {
+    let result = "";
+    switch (type.kind) {
+        case "LIST":
+            result += "[";
```
Comment (josemigallas) [inline ui/components/common/GraphQLFormatters.js]: Let's use template strings instead.

File: ui/components/common/GraphQLFormatters.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+const formatType = type => {
```
Comment (josemigallas) [inline ui/components/common/GraphQLFormatters.js]: Not a component, should be located out of this folder. (same goes for validators but.. too late)

File: ui/components/schema/TypeList.js [hunk 3]
```diff
@@ -133,26 +66,7 @@ class TypeList extends Component {
...
-                        return this.renderLoading();
-                    }
-                    if (error) {
-                        return this.renderError(error);
-                    }
-                    return this.renderList(data);
-                }}
-            </Query>
-        );
+        return this.renderList();
```
Comment (josemigallas) [inline ui/components/schema/TypeList.js]: What is this method for?

Comment (pb82) [reply ui/components/schema/TypeList.js]: Good question, i'll remove the call

**Pull request merged**